src/core/server/http/http_config.ts

```diff
@@ -31,6 +32,7 @@ const configSchema = schema.object(
         validate: match(validBasePathRegex, "must start with a slash, don't end with one"),
       })
     ),
+    gracefulShutdownTimeout: schema.duration({ defaultValue: '30s' }),
```

Comment:
shouldn't we set reasonable `min` and `max` limits? say, `min: 1sec`, `max: 2min.`

Reply:
Can you elaborate? This is a `timeout` (aka, max time to wait for the graceful shutdown to occur). If the server can finish earlier, it will. If it hits the 30s timeout, it will force-close the connections.

Reply:
right, but what happens if a user sets timeout `0sec`?

Reply:
oh! gotcha! Funny enough, `0sec` will wait 5s because of [hapi's implementation](https://github.com/hapijs/hapi/blob/master/lib/core.js#L395). I'll add `min: 1s` to ensure, at least, something. Thanks for suggesting it!